french/script/story/st_00.rpy

```diff
@@ -6,901 +6,901 @@ translate french s00_last_time_on_jpde_265b952a:
-    sk ""
+    sk "Tsk, qu'importe !{w=0.3} Ouvrez juste le chemin."
 
 # game/script/story/st_00.rpy:363
 translate french s00_birth_of_the_red_fang_bd32cead:
 
     # d "Good choice.{w=0.3} Now..."
-    d ""
+    d "Bonne pioche.{w=0.3} Maintenant..."
 
 # game/script/story/st_00.rpy:364
 translate french s00_birth_of_the_red_fang_cae334ad:
 
     # rf_2 sun "Engage them up close!{w=0.3} I will cover you!"
-    rf_2 sun ""
+    rf_2 sun "Attaquez-les de près !{w=0.3} Je vous couvre !"
 
 # game/script/story/st_00.rpy:365
 translate french s00_birth_of_the_red_fang_81d7ea9c:
 
     # rf_1 sun "You're dead, Sienna!"
-    rf_1 sun ""
+    rf_1 sun "T'es morte, Sienna !"
 
 # game/script/story/st_00.rpy:368
 translate french s00_birth_of_the_red_fang_35acae02:
 
     # d "I trust you won't be holding back?"
-    d ""
+    d "Je vous fais confiance pour ne pas vous retenir ?"
 
 # game/script/story/st_00.rpy:374
 translate french s00_birth_of_the_red_fang_2ed93a59:
 
     # sk ".{w=0.3}.{w=0.3}.{w=1.0}No.{w=0.5} I won't."
-    sk ""
+    sk ".{w=0.3}.{w=0.3}.{w=1.0}Non.{w=0.5} Aucunement."
 
 # game/script/story/st_00.rpy:412
 translate french battle_00_01_end_f38bd1a7:
 
     # rf_2 sun "UGH...!"
-    rf_2 sun ""
+    rf_2 sun "UGH... !"
 
 # game/script/story/st_00.rpy:413
 translate french battle_00_01_end_e7f0a17a:
 
     # rf_1 sun "Damn...{w=0.5} You..."
-    rf_1 sun ""
+    rf_1 sun "Allez...{w=0.5} Au diable..."
 
 # game/script/story/st_00.rpy:424
 translate french battle_00_01_end_aa14d17d:
 
     # sk nulla ".{w=0.3}.{w=0.3}.{w=0.5}you gods damned fools...{w=0.5} How could you be so stupid?!"
-    sk nulla ""
+    sk nulla ".{w=0.3}.{w=0.3}.{w=0.5} Bande d'idiots...{w=0.5} Comment vous pouvez être aussi stupides ?!" # "Bande de cons " ?
 
 # game/script/story/st_00.rpy:425
 translate french battle_00_01_end_f78e4a78:
 
     # d nulla "Well done.{w=0.3} But don't relax just yet..."
-    d nulla ""
+    d nulla "Bien joué.{w=0.3} Mais faut pas se relâcher tout de suite..."
 
 # game/script/story/st_00.rpy:426
 translate french battle_00_01_end_64c22de3:
 
     # adt sun neutral "This is quite enough.{w=0.3} I don't know who you are or what you're here for exactly, human..."
-    adt sun neutral ""
+    adt sun neutral "C'est déjà bien assez.{w=0.3} Je ne sais pas qui tu es ou pour quoi tu es là exactement, humain..."
 
 # game/script/story/st_00.rpy:432
 translate french battle_00_01_end_ffb62b58:
 
     # sk "Adam!"
-    sk ""
+    sk "Adamn !"
 
 # game/script/story/st_00.rpy:433
 translate french battle_00_01_end_88b0965d:
 
     # adt nulla "But Sienna dies here.{w=0.3} You've just added another corpse to the pile."
-    adt nulla ""
+    adt nulla "Mais Sienna meurt ici.{w=0.3} Tu as juste ajouté un autre cadavre au tas."
 
 # game/script/story/st_00.rpy:436
 translate french battle_00_01_end_88863b26:
 
     # d "Heh...{w=0.5} Sorry, no can do."
-    d ""
+    d "Eh...{w=0.5} Désolé, mais je n'y peux rien."
```

Comment:
"Désolé, mais je refuse." (no can do, c'est I won't do it -> je vais pas mourir ici merci bien)

Reply:
David en mode Julien Lepers: "C'est non." XD